module/dkg/broker.go

```diff
@@ -148,21 +138,25 @@ func (b *Broker) Broadcast(data []byte) {
 		}
 		maxedExpRetry := retry.WithMaxRetries(retryMaxPublish, expRetry)
 
+		clientIndex, dkgContractClient := b.getInitialContractClient()
+		onMaxConsecutiveRetries := func(totalAttempts int) {
+			clientIndex, dkgContractClient = b.updateContractClient(clientIndex)
+			b.log.Warn().Msgf("broadcast: retrying on attempt (%d) with fallback access node at index (%d)", totalAttempts, clientIndex)
+		}
+		afterConsecutiveFailures := retrymiddleware.AfterConsecutiveFailures(retryMaxConsecutiveFailures, maxedExpRetry, onMaxConsecutiveRetries)
+
 		attempts := 1
```

Comment:
do we need attempts only for logging?

Reply:
yes @tarakby requested we log the attempts when we were debugging DKG broadcast issues some time after retries were introduced.